packages/evolution-backend/src/scripts/auditInterviewsV2.task.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import 'chaire-lib-backend/lib/config/dotenv.config'; // Unused, but must be imported
```

Comment:
This task should be run in the survey context, as all the setup that is in the `server.ts` file, setting the server configuration, etc. must be done in this script's context.

Reply:
Il faut que ce soit roulé dans le projet od_mtl_2023 malheureusement. Nous n'avons pas accès à la config du projet ici. Il faudrait soit appeler le process du serveur (qui a la config) pour que le script roule là, ou je ne sais pas... Mais bon, la fonction dans interviews.ts, elle, est ben correcte ici.

Reply:
Is there something to change here, or we just aunch the task from the survey context?